test/test_long.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+#set -o errexit
+set -o nounset
+set -o pipefail
+
+# FROM SORT DOCUMENTATION
+#*** WARNING ***
+#The locale specified by the environment affects sort order.
+#Set LC_ALL=C to get the traditional sort order that uses
+#native byte values.
+
+wget wget https://en.wikipedia.org/wiki/Chess -qO long
```

Comment:
In this case I think you could just download `https://en.wikipedia.org/wiki/Chess` and store it inside the repo. Some of the pros:
- test is quicker
- no internet access needed
- no `wget` dependency
- if `wikipedia` is down your test is going to fail for a reason other than your code (search for "flaky tests")

Reply:
Okay, that is a good point, thank you